src/utils/useActivate.ts

```diff
@@ -0,0 +1,117 @@
+import { ipcRenderer } from "electron";
+import { NodeInfo } from "src/config";
+import { getSdk } from "src/generated/graphql-request";
+import { useTx } from "src/utils/useTx";
+import { useLoginSession } from "./useLoginSession";
+import { useStore } from "./useStore";
+
+type ActivationStep =
+  | "getActivationInfo"
+  | "getGraphQLClient"
+  | "getActivationAddress"
+  | "createActivateAccountTx"
+  | "stageTx";
+
+type ActivationSuccessResult = {
+  result: true;
+  txId: string;
+};
+
+type ActivationFailResult = {
+  result: false;
+  error: {
+    error: unknown;
```

Comment:
Could we specify a more specified type instead of `unknown`?

Reply:
The error is not thrown by internal code, so it's hard to determine the type of the error.
Also, the error instance is the same with the error instance that has been catched, so it's original type is `unknown` too.

Reply:
@longfin Is this a major issue that blocks merge? If it isn't, can I consider it resolved and continue?

Reply:
Fixed on [3bbe211](https://github.com/planetarium/9c-launcher/pull/2110/commits/3bbe211c71e92e0c7b58b5b8f34d8aa946b796b3)